mysql/Positive-Technologies/MySqlLexer.g4

```diff
@@ -648,6 +648,7 @@ RELOAD:                              'RELOAD';
 SHUTDOWN:                            'SHUTDOWN';
 SUPER:                               'SUPER';
 PRIVILEGES:                          'PRIVILEGES';
+SESSION_VARIABLES_ADMIN:             (SINGLE_QUOTE_SYMB | DOUBLE_QUOTE_SYMB | REVERSE_QUOTE_SYMB)? 'SESSION_VARIABLES_ADMIN' (SINGLE_QUOTE_SYMB | DOUBLE_QUOTE_SYMB | REVERSE_QUOTE_SYMB)?;
```

Comment:
It makes sense to use `fragment` rule here to get rid of duplication:

```antlr
SESSION_VARIABLES_ADMIN: QUOTE_SYMB? 'SESSION_VARIABLES_ADMIN' QUOTE_SYMB?;

fragment QUOTE_SYMB
    : SINGLE_QUOTE_SYMB | DOUBLE_QUOTE_SYMB | REVERSE_QUOTE_SYMB
    ;
```

Reply:
@KvanTTT Fixed, thanks for the tip!